### PR TITLE
docs: provide a more accurate example for interacting with custom fields

### PR DESCRIPTION
**Pull request by sherwinski**

### CODE REVIEW

**Comment by sherwinski** [comment]
@jayeb Mind reviewing the new example for updating custom fields? I wanted to address the point @daletom raised about how existing fields need to be included when updating or else they will be overwritten.

File: README.md [hunk 1]
```diff
@@ -85,35 +86,50 @@ imgix.request(`assets/${sourceId}/${originPath}`)
...
+    /*
+    ** Populate `document` with all pre-existing fields
+    ** so as to not overwrite them when sending a PATCH request
+    **
+    ** `id` and `type` must be included in the request as well
+    */
+    document.data.attributes.custom_fields = response.data.attributes.custom_fields;
+    document.data.attributes.custom_fields.type = 'dessert';
+    document.data.id = response.data.id;
+    document.data.type = response.data.type;
```
Comment (jayeb) [inline README.md:121]: I think defining these two attributes here kind of muddies the waters a bit. In a real-world scenario, you'd know the ID and type of the object you're planning to patch, so why not just include them up on L104-108 where the `document` structure is being built?

Comment (sherwinski) [reply README.md:121]: Out of curiosity, can `type` ever be anything other than `assets`?

Comment (jayeb) [reply README.md:121]: If you're hitting a non-asset endpoint, yes! Sources use `type: "sources"`, users use `type: "users"`, etc. etc.

File: README.md [hunk 2]
```diff
@@ -85,35 +86,48 @@ imgix.request(`assets/${sourceId}/${originPath}`)
...
+
+### Update custom fields for a single image
+
+```js
+var document = {
+    'data': {
+        'attributes': {}
+    },
+    'id': `${sourceId}/uploads/pecanpie.jpg`,
+    'type': 'assets'
```
Comment (jayeb) [inline README.md:109]: Little mixup with the data structure here.

```suggestion
    'data': {
        'id': `${sourceId}/uploads/pecanpie.jpg`,
        'type': 'assets',
        'attributes': {}
    }
```

**Pull request merged**